benchmark/CMakeLists.txt

```diff
@@ -32,18 +32,44 @@ target_link_libraries(ubench
 	pthread
 	m)
 
+if (UMF_BUILD_BENCHMARKS_MT)
+	add_executable(multithread_bench multithread.cpp)
+	target_link_libraries(multithread_bench
+		umf
+		${LIBS_OPTIONAL}
+		pthread
```

Comment:
do we need pthread?

Reply:
Yes, eventually, we should switch to `Threads::Threads` target but we don't do that anywhere for now.